front/lib/iam/session.ts

```diff
@@ -135,7 +136,8 @@ export function makeGetServerSidePropsRequirementsWrapper<
 
       if (
         requireCanUseProduct &&
-        !auth?.subscription()?.plan.limits.canUseProduct
+        (!auth?.subscription()?.plan.limits.canUseProduct ||
```

Comment:
I completely forgot about the canUseProduct, I think we should remove this change and update the plan in db instead to set it to false for this plan. WDYT?

Reply:
Sounds good, having `canUseProduct` to true is indeed weird